README.md

```diff
@@ -26,11 +26,18 @@ See the language-specific guides for further details on support and usage of the
 
 Further information on the Azure SQL binding for Azure Functions is also available in the [Azure Functions docs](https://docs.microsoft.com/azure/azure-functions/functions-bindings-azure-sql).
 
+## Supported SQL Server Versions
+
+This extension uses the [OPENJSON](https://learn.microsoft.com/sql/t-sql/functions/openjson-transact-sql) statement which requires SQL Server 2016 or higher and a database compatibility level of 130+. Older versions will currently fail to execute properly.
+
+For more information about options to upgrade versions see the [SQL Server end of support options](https://learn.microsoft.com/sql/sql-server/end-of-support/sql-server-end-of-support-overview) page.
+
+Azure SQL Databases are fully supported.
```

Comment:
@dzsquared is "Azure SQL Databases" what we should be calling the Azure SQL suite of products?

Reply:
our triad of 3 products that we support is mentioned in my comment above - "Azure SQL Database" is 1 product, then "Azure SQL Managed Instance" is another.
I'd prefer to avoid lumping it as "Azure SQL" lest it opens confusion to whether we support Synapse, which we do not.

Reply:
Yeah - to be clear this is only supposed to be for Azure SQL Database. I haven't had a chance to test out MI yet so wanted to hold off on saying anything until I can verify it (although it should work according to the docs). 

So given that do you think leaving it as Azure SQL Databases is fine or should I be more specific? (or even hold off until I can test MI so we can have an accurate list of supported engines?)

Reply:
Updated it with the suggestion Drew had above - I'll test out MI before getting this in so that we have everything here be complete.